packages/edit-site/src/components/layout/index.js

```diff
@@ -185,10 +180,6 @@ export default function Layout() {
 						'is-full-canvas': canvasMode === 'edit',
 						'has-fixed-toolbar': hasFixedToolbar,
 						'is-block-toolbar-visible': hasBlockSelected,
-						'has-block-breadcrumbs':
```

Comment:
We need to keep the `has-block-breadcrumbs` class to avoid a regression for [this one](https://github.com/WordPress/gutenberg/issues/53198)

Reply:
We've discussed recently with @jameskoster that the snackbars should remain in the exact same position regardless of what the UI is showing. It's already the case in trunk and this class is not used at all in our styles.

Reply:
Thank you. Is there any place I can see the discussion around the topic?

Reply:
Thank you!